nca/NetworkConfig/NetworkConfigQuery.py

```diff
@@ -1166,37 +1468,83 @@ def get_query_type():
     def exec(self):
         query_answer = self.is_identical_topologies(True)
         if query_answer.bool_result and query_answer.output_result:
-            return query_answer  # identical configurations (contained)
+            return query_answer, None, None  # identical configurations (contained)
 
         contained_1_in_2 = ContainmentQuery(self.config1, self.config2, self.output_config).exec()
         contained_2_in_1 = ContainmentQuery(self.config2, self.config1, self.output_config).exec()
-        explanation_not_contained_self_other = \
-            contained_1_in_2.output_result + ':' + contained_1_in_2.output_explanation
-        explanation_not_contained_other_self = \
-            contained_2_in_1.output_result + ':' + contained_2_in_1.output_explanation
         if contained_1_in_2.bool_result and contained_2_in_1.bool_result:
             return QueryAnswer(bool_result=True,
                                output_result=f'The two network configurations {self.name1} and {self.name2} '
                                              'are semantically equivalent.',
-                               numerical_result=3)
+                               numerical_result=3), None, None
+
+        sub_query_answer1 = QueryAnswer(output_result=contained_1_in_2.output_result,
+                                        output_explanation=contained_1_in_2.output_explanation)
+        sub_query_answer2 = QueryAnswer(output_result=contained_2_in_1.output_result,
+                                        output_explanation=contained_2_in_1.output_explanation)
```

Comment:
Why do you have to recreate `QueryAnswer `objects here and cannot use the objects returned from running `exec `on `ContainmentQuery`?

Reply:
actually I can use them, I just need the `output_result` and `output_explanation` of them for 'computing' the `query_output`. I thought that way is easiest to read and understand the "purpose" of them.
I will simplify this (by not creating new objects)